app/components/base/selectable-list/list-or-table-base.ts

```diff
@@ -10,17 +10,18 @@ export class ListOrTableBase extends SelectableList implements OnDestroy {
         if (table && this._table !== table) {
             this._table = table;
             this._clearTableSubs();
-            this._tableSubs.push(table.selectedItemsChange.subscribe((items) => {
-                this.selectedItemsChange.emit(items);
-                this.selectedItems = items;
-            }));
+            // this._tableSubs.push(table.selectedItemsChange.subscribe((items) => {
+            //     this.selectedItemsChange.emit(items);
+            //     this.selectedItems = items;
+            // }));
 
-            this._tableSubs.push(table.activatedItemChange.subscribe((event) => {
-                if (!(event.initialValue && this.activatedItem)) {
-                    this.activatedItemChange.emit(event);
-                    this.activatedItem = event.key;
-                }
-            }));
+            // TODO-Tim check dis
+            // this._tableSubs.push(table.activatedItemChange.subscribe((event) => {
+            //     if (!(event.initialValue && this.activatedItem)) {
+            //         this.activatedItemChange.emit(event);
+            //         this.activatedItem = event.key;
+            //     }
+            // }));
         }
```

Comment:
check this and above commented out code as well. 
just adding comments so they don't get lost in the noise.